functions/helloworld/package.json

```diff
@@ -22,10 +22,11 @@
   },
   "dependencies": {
     "@google-cloud/debug-agent": "1.0.0",
+    "pug": "2.0.0-beta.12",
     "safe-buffer": "5.0.1"
   },
   "devDependencies": {
-    "@google-cloud/nodejs-repo-tools": "1.3.1",
+    "@google-cloud/nodejs-repo-tools": "1.3.2",
```

Comment:
FYI: you'll need to revert this change. (It will cause a merge conflict with #370.)